public/less/_user.less

```diff
@@ -20,6 +20,9 @@
                         padding: 10px;
                         list-style: none;
 
+                        overflow: scroll; // prevent contents from spilling over
+                        display: flex; // lay out contents horizontally
```

Comment:
This also make the orgs image disapear (at least on default theme).

For comparison with previous image without change we see the org image.
![image](https://user-images.githubusercontent.com/4052400/59809066-46950d80-92ff-11e9-80e5-aeee53734b27.png)